### PR TITLE
Add new bad words, improve README and other stuff

### PR DESCRIPTION
**Pull request by voidpy**

### CODE REVIEW

File: unswear.user.js [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
 (function() {
-  // Change this to false if you want specific badWords to be replace with their goodWords
-  // but the goodWords must have the same lenght as badWords
+  // Change randomMode to false if you want specific badWords to be replaced with their goodWords.
+  // If randomMode's false, goodWords must have the same length as badWords. 
   let randomMode = true;
 
   let goodWords = ["❤️", "🎀"];
-  let badWords = ["fuck"];
+  let badWords = ["fuck(\w+)?", "motherfuck(\w+)?", "shit(\w+)?", "bitch(\w)?"];
```
Comment (devTeaa) [inline unswear.user.js]: Do we really need the (\w+)? as I didn't see much difference on it
```suggestion
  let badWords = ["fuck", "motherfuck", "shit", "bitch"];
```

**Review by devTeaa — CHANGES_REQUESTED** [review]
hi @voidpy thanks for the docs update! It looks cleaner now, but I have some small comments, let me know what you think

File: unswear.user.js (hunk 1 above)
Comment (voidpy) [reply unswear.user.js]: With "(\w+)?", regex can match words like "fucking", "fuckity", "motherfucking" and-the-like. It means something like "Match 'fuck'. The word can contain optional characters after the last character -- In this case, 'k' --." This way, we can filter more bad words without hesitating too much (thanks to regular expressions ^^). I hope I could explain myself (I'm not pretty good at it. lol).

Comment (devTeaa) [reply unswear.user.js]: Oh that makes more sense, I didn't notice the changes Thanks for the PR

**Review by devTeaa — APPROVED** [review]
Code changes explained

**Pull request merged**